src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/widgets/SchedulingAndDescription/index.tsx

```diff
@@ -93,12 +93,9 @@ export const SchedulingAndDescription = () => {
     (updatedVideoProperty: Partial<Video>) => {
       videoMutation.mutate(updatedVideoProperty);
     },
+    1000,
```

Comment:
I don't think this is a proper fix, this would just reduce the risk for it to happen expecting user to type fast enough for not stopping for one second and then type again.
A better fix would probably to trigger changes (and toast) only if after the update, the test arrea content has not changed.

Reply:
Actually, the fix is not here, the fix is by removing the useEffect. I added some times just too have less Toast jumping on each others, 500ms was a bit short when you tape.